From 72d5c4c3f7d2a5d329eb74fa1d1f69c1fb8c15b9 Mon Sep 17 00:00:00 2001
From: Petr Mladek <pmladek@suse.cz>
Date: Mon, 18 Jul 2011 16:50:18 +0200
Subject: [PATCH 3/3] distro-install: fix en-US-only installation

generate en-US file list when --with-distro=""

sigh, we should set WITH_LANG=en-US when no language is selected; the empty
string is pretty ugly; unforrunately, many makefile tests check for
this empty variable
---
 configure.in |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/configure.in b/configure.in
index 18f93d2..65813cb 100755
--- a/configure.in
+++ b/configure.in
@@ -7906,6 +7906,9 @@ for lang in $WITH_LANG ; do
 done
 # list with substituted ALL
 WITH_LANG_LIST=`echo $WITH_LANG | sed "s/ALL/$ALL_LANGS/"`
+# this variable is used only by bin/distro-install-* helper scripts
+# they need a real list of languages
+test -z "$WITH_LANG_LIST" && WITH_LANG_LIST="en-US"
 AC_SUBST(ALL_LANGS)
 AC_SUBST(WITH_LANG)
 AC_SUBST(WITH_LANG_LIST)
-- 
1.7.3.4

